app/routes/create-catch-certificate/$catchCertificate/what-are-you-exporting/whatAreYouExporting.ts

```diff
@@ -71,14 +71,13 @@ export const getAddedSpeciesPerUser = async (catchCertificate?: string): Promise
 export const getSpecies = async (): Promise<ISpecies[]> => {
   const response: Response = await fetch(SPECIES_URL);
   const species = await response.json();
-
   return species;
 }
 
 export const getStates = async (): Promise<IStates[]> => {
   const response: Response = await fetch(STATES_URL);
   const states = await response.json();
-
+  console.log(states);
```

Comment:
@daniele-zurico `console.log(states);`

Reply:
because you folks do bad things on the server :) .... removed